common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -624,3 +624,307 @@ def test_upload_course_video_thumbnail_image(self):
         file_to_upload = 'image.jpg'
         self.settings_page.upload_image('#upload-video-thumbnail-image', file_to_upload)
         self.assertIn(file_to_upload, self.settings_page.get_uploaded_image_path('#video-thumbnail-image'))
+
+
+class CourseSettingsTest(StudioCourseTest):
+    """
+    Class to test course settings.
+    """
+    COURSE_START_DATE_CSS = "#course-start-date"
+    COURSE_END_DATE_CSS = "#course-end-date"
+    ENROLLMENT_START_DATE_CSS = "#course-enrollment-start-date"
+    ENROLLMENT_END_DATE_CSS = "#course-enrollment-end-date"
+
+    COURSE_START_TIME_CSS = "#course-start-time"
+    COURSE_END_TIME_CSS = "#course-end-time"
+    ENROLLMENT_START_TIME_CSS = "#course-enrollment-start-time"
+    ENROLLMENT_END_TIME_CSS = "#course-enrollment-end-time"
+
+    course_start_date = '12/20/2013'
+    course_end_date = '12/26/2013'
+    enrollment_start_date = '12/01/2013'
+    enrollment_end_date = '12/10/2013'
+
+    dummy_time = "15:30"
+
+    def setUp(self, is_staff=False, test_xss=True):
+        super(CourseSettingsTest, self).setUp()
+
+        self.settings_page = SettingsPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+        # Before every test, make sure to visit the page first
+        self.settings_page.visit()
+        self.ensure_input_fields_are_loaded()
+
+    def set_course_dates(self):
+        """
+        Set dates for the course.
+        """
+        dates_dictionary = {
+            self.COURSE_START_DATE_CSS: self.course_start_date,
+            self.COURSE_END_DATE_CSS: self.course_end_date,
+            self.ENROLLMENT_START_DATE_CSS: self.enrollment_start_date,
+            self.ENROLLMENT_END_DATE_CSS: self.enrollment_end_date
+        }
+
+        self.settings_page.set_value(dates_dictionary)
+
+    def ensure_input_fields_are_loaded(self):
+        """
+        Ensures values in input fields are loaded.
+        """
+        EmptyPromise(
+            lambda: '' != self.settings_page.q(css='#course-organization')[0].get_attribute('value'),
```

Comment:
The empty string evaluates to boolean False. So you should not need to compare this explicitly here.
Reference: https://www.python.org/dev/peps/pep-0008/#programming-recommendations, see the section that says 

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.